colossalai/fx/passes/meta_info_prop.py

```diff
@@ -22,6 +22,12 @@ class TensorMetadata(NamedTuple):
     # behaviour by appending sharding spec into list.
 
 
+@compatibility(is_backward_compatible=False)
+class node_size(NamedTuple):
```

Comment:
You should name it as NodeSize. It is a class name!

Reply:
I actually adopted this lower-case convention from https://github.com/pytorch/pytorch/blob/master/torch/fx/passes/graph_manipulation.py. Anyway, I removed this class because we won't be using it anymore.